app/controllers/batch_contexts_controller.rb

```diff
@@ -15,9 +15,10 @@ def new
     )
   end
 
-  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
   def create
     params = batch_contexts_params
+    params[:ocr_languages] ||= []
 
```

Comment:
is this because it will be nil if not entered at all and we want an empty array?

Reply:
Correct.